src/middlewares/authJWT.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require('jsonwebtoken')
+const User = require('../models/User/UserSchema')
+
+const verifyToken = (req,res,next)=>{
+  if(req.headers && req.headers.authorization && req.headers.authorization.split(' ')[0]==="JWT"){
+   jwt.verify(req.headers.authorization.split(' ')[1], process.env.JWT_SECRET , (err, decode)=>{
+    if(err){
+      req.user = undefined
+      res.status(500).send({message: err.message})
```

Comment:
Should return

Reply:
return as in ending the fn call here and just return ?

Reply:
return as in ending the fn call here and just return ?